tests/power_distribution/test_power_distributor.py

```diff
@@ -33,6 +33,7 @@
 )
 
 T = TypeVar("T")  # Declare type variable
+SAFETY_TIMEOUT = 10.0
```

Comment:
1. Maybe you could move that to `conftest.py`? 
Then it could be used in many unit tests.

2. Could you add comment and explain why we use this SAFETY_TIMEOUT? This is clear now, but won't be in the future or for new users.